spec/v1.0-draft/field_types.md

```diff
@@ -46,6 +46,50 @@ _DigestSet (object)_
 > -   `{"sha256": "abcd"}` does not match `{"sha256": "fedb", "sha512": "abcd"}`
 > -   `{"somecoolhash": "abcd"}` uses a non-predefined algorithm
 
+<a id="Reference"></a>
+_Reference (object)_
+
+> A size-efficient representation of any hashable artifact referenced by
+> another in-toto attestation.
+>
+> Schema:
+>
+>```jsonc
+> {
+>   "localName": "<NAME>",
+>   "digest": { "<ALGORITHM>": "<HEX VALUE>", ... },
+>   "downloadLocation": "<RESOURCE URI>", // optional
+>   "mimeType": "<MIME TYPE>" // optional
+> }
+>```
+>
+> Fields:
+>
+> -   `localName` (_string, required_): Human-readable identifier to
+>     distinguish the referenced artifact locally. The semantics are up to
+>     the producer and consumer.
+> -   `digest` (_[DigestSet], required_): The producer and consumer must
+>     agree on acceptable algorithms.
+> -   `downloadLocation` (_[ResourceURI], optional_): The location of the
+>     referenced artifact. To enable automated downloads by consumers, the
+>     specified location SHOULD be resolvable.
+> -   `mimeType` (_string, optional_): The [MIME Type][] (i.e., media type)
+>     of the referenced artifact.
+>
+> Note: A reference is bound to the `digest` field. All other fields are
+> informational.
+>
+> Example:
+>
+> ```jsonc
+> { 
+>   "localName": "gcc9.3.0-rebuilderd-attestation",
+>   "digest": { "sha256": "abcdabcde..." },
+>   "downloadLocation": "http://example.com/rebuilderd-instance/gcc_9.3.0-1ubuntu2_amd64.att",
+>   "mimeType": "application/vnd.in-toto+json"
```

Comment:
optional: WDYT about renaming this 'mediaType' to match SLSA's [ArtifactReference](https://github.com/slsa-framework/slsa/blob/main/docs/provenance/v1.md#artifactreference) ?

Reply:
I'm fine changing it to be consistent.